transpiler/src/target/typescript/helpers/getTargetFileDestination.ts

```diff
@@ -152,6 +153,7 @@ const getTargetFileDestination = (
 const getFilePathRelativeToModule = (
```

Comment:
This function returns the path relative to module start
e.g. `application/handlers/HandlerName`.
Maybe cross bounded context imports should be handled by the one who calls this function if possible?
(Only in the case of Integration-Event)
Otherwise this function needs renaming i think